front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/containers/index.tsx

```diff
@@ -117,34 +113,11 @@ const HomepageBuilderPage = () => {
               platformLocale={locale}
             >
               <Box width="1000px">
-                <ContentBuilderFrame editorData={getEditorData()}>
-                  <HomepageBanner
-                    homepageSettings={{
```

Comment:
This was not doing anything, not sure why it was even here. The editor data was always defined